### DISCUSSION TITLE
Need help to setup new project (Facing issue - Plugin [id: 'io.invertase.gradle.build', version: '1.3'] was not found...)

### DISCUSSION THREAD

**Discussion by rchowdhe** [invertase/react-native-firebase#4436] [Q&A]
I followed below steps to integrate  `react-native-firebase` into my app.

1. Initiated (create) new react native app using react native cli `react-native init OCRFirebaseApp`
2. Executed command `react-native run-android` to check if app is running or not after setup, It started successfully in android device and I can see "Welcome to React" page.
3. Now executed command `npm install --save @react-native-firebase/app` under project root to integrated firebase plugin, and it completed successfully.
4. But now after doing some necessary adoption as mentioned here [https://rnfirebase.io/](https://rnfirebase.io/) when i again execute the command `react-native run-android` to run the application, it throws below error:

**Can someone please suggest what i missed or doing something wrong?**

```
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\DEVELOPMENT\OCR\OCRFirebaseApp\node_modules\@react-native-firebase\app\android\build.gradle' line: 18

* What went wrong:
Plugin [id: 'io.invertase.gradle.build', version: '1.3'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'io.invertase.gradle.build:io.invertase.gradle.build.gradle.plugin:1.3')
  Searched in the following repositories:
    Gradle Central Plugin Repository
```

![image](https://user-images.githubusercontent.com/23568939/96869733-82109080-148d-11eb-8c58-a0482b74d132.png)

**Comment by mikehardy** [comment]
Strange... this works at least on linux and macOS: https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh (it may not work on Windows, sorry)

Here it is though, it turns up on web search, it should be available to gradle: https://plugins.gradle.org/plugin/io.invertase.gradle.build

**Reply by rchowdhe** [reply]
It means there is no way to continue react native development with integration of firebase on windows? if it so then I am really disappointed 😞

**Reply by mikehardy** [reply]
There definitely should be - though as react-native implies building for ios which implies a dependency on macOS I believe it just hasn't come up much! There is probably some extremely trivial configuration issue or script issue that - once resolved - will have everything working. You might try using plugin 1.4 ? it is current

**Reply by rchowdhe** [reply]
Yes I have already tried with 1.4, but no success, same error message.
